pkg/chunked/filesystem_linux.go

```diff
@@ -58,7 +63,7 @@ func copyFileContent(srcFd int, fileMetadata *fileMetadata, dirfd int, mode os.F
 	src := fmt.Sprintf("/proc/self/fd/%d", srcFd)
 	st, err := os.Stat(src)
 	if err != nil {
-		return nil, -1, fmt.Errorf("copy file content for %q: %w", destFile, err)
+		return nil, -1, err
```

Comment:
Not sure this is an improvement on the error. 
Now we will get "failed to stat /proc/self/fd/%d" Rather then destFile?

Reply:
I've dropped the patch from this PR.

I will rework errors later